examples/https.js

```diff
@@ -1,10 +1,11 @@
 'use strict'
 
 const fs = require('fs')
+const path = require('path')
 const fastify = require('../fastify')({
   https: {
-    key: fs.readFileSync('../test/https/fastify.key'),
-    cert: fs.readFileSync('../test/https/fastify.cert')
+    key: fs.readFileSync(path.join(__dirname, '../test/https/fastify.key')),
```

Comment:
path.resolve?

Reply:
why?  `__dirname` is absolute.

Reply:
Yeah but `path.resolve('test/https/fastify.key')` is nicer

Reply:
That is not correct. Because `resolve` assumes the file is run from the root of the project.